app/helpers/sellers_helper.rb

```diff
@@ -2,7 +2,7 @@
 
 module SellersHelper
   def self.generate_seller_json(seller:)
-    locations = seller.locations
+    locations = seller.locations[0]
```

Comment:
also do you mind logging Rails.logger.info whenever seller.locations.size is > 1? It'll be useful information for us to see when a seller has more than one location so that we can go and delete the extra location